doc/UserGuide.md

```diff
@@ -35,6 +35,8 @@ Format: `help`
 ### Adding a person: `add`
 Adds a person to the address book<br>
 Format: `add NAME [p]p/PHONE_NUMBER [p]e/EMAIL [p]a/ADDRESS [t/TAG]...` 
+
+ADDRESS = BLOCK, STREET, UNIT, POSTALCODE
```

Comment:
Great! It's nice that you also update the user guide.
May be you could give more information about the format of address (e.g. `Address must be of the following format:`) rather then just giving the equation `ADDRESS = BLOCK, STREET, UNIT, POSTALCODE`